introduction/java_loops_II

```diff
@@ -2,24 +2,30 @@ import java.util.*;
 import java.io.*;
 import java.lang.Math;
 
-class Solution{
-  public static void main(String []argh) {
-    Scanner in = new Scanner(System.in);
-      int t = in.nextInt();
+class Solution {
+
+  statis private final int EXPONENT = 2;
+  
+  public statis void loops(int firstNumber, int secondNumber, int power, int temporaryResult) {
```

Comment:
typo: statis.
Please run your code before submission. Do not send code that doesn't compile.